app/assets/javascripts/discourse/app/controllers/user-topics-list.js

```diff
@@ -105,8 +82,8 @@ export default Controller.extend(BulkTopicSelection, {
 
   @action
   showInserted() {
-    this.model.loadBefore(this.newIncoming);
-    this._resetTracking();
+    this.model.loadBefore(this.pmTopicTrackingState.newIncoming);
+    this.pmTopicTrackingState.resetTracking();
```

Comment:
why don't we check pmTopicTrackingState is not null here ? Is it intentional to think at this point it can't be null ?

Reply:
Yup this action is only called when the info is displayed. In order for the info to be displayed, `pmTopicTrackingState` will not be `null`.